docs/integrations/destinations/databricks.md

```diff
@@ -9,6 +9,11 @@ This destination syncs data to Delta Lake on Databricks Lakehouse. Each stream i
 You **must** be using Unity Catalog to use this connector.
 :::
 
+:::info
+OAuth2 authentication is only supported in AWS deployments. If you are running
```

Comment:
Please note, at this time OAuth2 authentication is only supported in AWS deployments. If you are running
Databricks in GCP, you **must** use an access token.